cdk/database/__init__.py

```diff
@@ -62,3 +62,43 @@ def dynamodb_single_table(self):
                                         partition_key=aws_dynamodb.Attribute(
                                             name='PK',
                                             type=aws_dynamodb.AttributeType.STRING))
+
+    def dynamodb_dual_tables(self):
```

Comment:
Consider changing this to two table specific functions

`dynamodb_visits_table`, and `dynamodb_users_table`. We may look to add new tables in the future e.g. `dynamodb_surveys_table` and it will be painful to have to change the name of the function each time.